src/libAtomVM/interop.h

```diff
@@ -65,6 +67,7 @@ term interop_map_get_value_default(GlobalContext *glb, term map, term key, term
 
 NO_DISCARD InteropFunctionResult interop_iolist_size(term t, size_t *size);
```

Comment:
I would rename these as well and add static inline aliases for backward compatibility.

Reply:
Same as above comment.